### PR TITLE
Docs: add Tetragon to the Roadmap and remove GSoD

### PR DESCRIPTION
**Pull request by xmulligan**
Please ensure your pull request adheres to the following guidelines:

```release-note
Added Tetragon to the roadmap
```

### CODE REVIEW

File: Documentation/community/roadmap.rst [hunk 1]
```diff
@@ -33,16 +33,17 @@ Major Feature Status
 +-----------------------------+------------------------------------------------------------+
 | Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
 +-----------------------------+------------------------------------------------------------+
-
+| Tetragon security           | Beta                                                       |
```
Comment (xmulligan) [inline Documentation/community/roadmap.rst]:
```suggestion
| Tetragon security           | Beta                                                       |

```

File: Documentation/community/roadmap.rst [hunk 2]
```diff
@@ -33,16 +33,18 @@ Major Feature Status
 +-----------------------------+------------------------------------------------------------+
 | Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
 +-----------------------------+------------------------------------------------------------+
+| Tetragon security           | Beta                                                       |
 
++-----------------------------+------------------------------------------------------------+
```
Comment (xmulligan) [inline Documentation/community/roadmap.rst:37]:
```suggestion
+-----------------------------+------------------------------------------------------------+

```

**Comment by xmulligan** [comment]
I think I finally figured out the correct formatting 🙃

File: Documentation/community/roadmap.rst [hunk 3]
```diff
@@ -33,6 +33,8 @@ Major Feature Status
 +-----------------------------+------------------------------------------------------------+
 | Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
 +-----------------------------+------------------------------------------------------------+
+| Tetragon security           | Beta                                                       |
```
Comment (qmonnet) [inline Documentation/community/roadmap.rst]:
```suggestion
| Tetragon Security           | Beta                                                       |
```

File: Documentation/community/roadmap.rst [hunk 4]
```diff
@@ -146,14 +148,16 @@ CI Test Improvements
 We have a comprehensive set of tests running in CI, but several contributors are
 currently working on `CI improvements`_ to make these more reliable and easier to
 maintain. This is a good area to get involved if you are interested in learning
-more about Cilium internals and development. 
+more about Cilium internals and development.
+
+Tetragon Security
+~~~~~~~~~~~~~~~~~
+
+Tetragon provides security observability through the JSON events and the Tetragon
```
Comment (qmonnet) [inline Documentation/community/roadmap.rst]: Security observability... and runtime enforcement, too?

File: Documentation/community/roadmap.rst [hunk 5]
```diff
@@ -146,14 +148,16 @@ CI Test Improvements
...
+CLI for things like process execution, file access, network observability, and
+privileged execution.
 
-Documentation Improvements
-~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We would like to make this Cilium documentation :ref:`easier to
-navigate<season_of_docs>`, with more example configurations and links to
```
Comment (qmonnet) [inline Documentation/community/roadmap.rst:155]: Do we want to keep the GSoD dedicated page at all?

**Review by qmonnet — CHANGES_REQUESTED** [review]
Looks good, with some minor comments below. Thanks!

File: Documentation/community/roadmap.rst (hunk 4 above)
Comment (xmulligan) [reply Documentation/community/roadmap.rst]: Agreed, I'll add that

Comment (xmulligan) [inline Documentation/community/roadmap.rst]:
```suggestion
Tetragon provides security observability and runtime enforcement through the JSON events and the Tetragon
```

File: Documentation/community/roadmap.rst (hunk 5 above)
Comment (xmulligan) [reply Documentation/community/roadmap.rst:155]: for now, yes https://github.com/cilium/cilium/pull/21338#discussion_r974247249

**Review by qmonnet — APPROVED** [review]

**Comment by qmonnet** [comment]
image

Let's make checkpatch happy (or just ditch this tag, I didn't author anything significant on this commit)

All good to me after that

**Comment by xmulligan** [comment]
@qmonnet done :)

**Comment by joestringer** [comment]
Weird, I'm not sure why this PR is hitting https://github.com/cilium/cilium/issues/21410 now but that failure is unrelated to the PR here. That is then causing a couple of other failures in the CI jobs. We can safely ignore those for this PR.

**Comment by xmulligan** [comment]
What could go wrong?

**Comment by joestringer** [comment]
@xmulligan ah, just as a heads up, when you create new PRs it would be worthwhile to first update your local tree to the same as cilium/cilium master. I believe that a `git pull --rebase` should be sufficient. This PR was based on top of some version from July, which is what caused #21410 due to some differences between the code in the PR here vs. the workflow tests.

**Pull request merged**